bot/utility/utility.py

```diff
@@ -95,6 +96,34 @@ async def about(self, ctx: commands.Context):
         embed.add_field(name="Links:", value=str_links)
         await ctx.send(embed=embed)
 
+    @commands.command(name='howto')
+    @command_log
+    async def howto(self, ctx: commands.Context, what: Optional[str] = None):
```

Comment:
"what" is not a very descriptive name. you could call it "topic" or sth similar.